### ISSUE TITLE
Dropout on Attention

### ISSUE THREAD

**Issue by MilesQLi** [jadore801120/attention-is-all-you-need-pytorch#72]
Hi, I find you apply dropout to attention weight in ScaledDotProductAttention. It seems that is not in the original paper?

**Comment by jadore801120** [comment]
It is used in the tensorflow implementation.
Hope it helps.
